README.md

```diff
@@ -19,11 +19,17 @@ semantic engine for the Ada programming language.
 Dependencies
 ------------
 
-To use Langkit:
+To use Langkit, you will need:
 
-* A Python 3.7+ intererpter. Python2 is no longer supported.
-* The mako template system for Python (see `REQUIREMENTS.dev`).
-* Clang-format.
+* A Python 3.7+ interpreter. Python2 is no longer supported.
+* Some python libraries, including the mako template system for Python (see
+  `REQUIREMENTS.dev` for the full list).
+* A recent version of the GNAT Ada compiler, either from your OS's packages, or
+  from [here](https://www.adacore.com/download).
+* The [gnatcoll-core](https://github.com/AdaCore/gnatcoll-core) and
+  [gnatcoll-bindings](https://github.com/AdaCore/gnatcoll-bindings) Ada
+  libraries.
```

Comment:
Let’s be more specific for gnatcoll-bindings, as not all bindings are useful and building all bindings bring other dependencies. I suggest:
```
* The Ada bindings for GMP and Libiconv from [gnatcoll-bindings](…) .
```